src/LieTheory/Types.jl

```diff
@@ -87,9 +87,9 @@ mutable struct DualRootSpaceElem
 
   `vec` must be a row vector of the same length as the rank of `R`.
   """
-  function DualRootSpaceElem(root_system::RootSystem, vec::QQMatrix)
-    @req size(vec) == (1, rank(root_system)) "Invalid dimension"
-    return new(root_system, vec)
+  function DualRootSpaceElem(R::RootSystem, vec::QQMatrix)
+    @req size(vec) == (1, rank(R)) "Invalid dimension"
+    return new(R, vec)
```

Comment:
I made this change to align it with the `RootSpaceElem` constructor (to verify they are identical) and also to match the docstring.